post-install.yml

```diff
@@ -68,12 +68,19 @@
 
     - name: Post Install Tasks | Tsuru key-list
       shell: >
-        tsuru key-list
+        tsuru key-list -n
       register: tsuru_keys
+    - name: Post Install Tasks | Get public key content
+      shell: cat ~/.ssh/tsuru_deployer.pub
+      register: public_key_content
+    - name: Post Install Tasks | Tsuru remove old key
+      shell: >
+        tsuru key-remove -y tsuru_deployer
+      when: "not public_key_content.stdout in tsuru_keys.stdout and 'tsuru_deployer' in tsuru_keys.stdout"
```

Comment:
This condition feels a bit loose. What do you think of removing the `and 'tsuru_deployer' in tsuru_keys.stdout` part and instead adding `| awk '$2 == "tsuru_deployer"'` to the `tsuru key-list -n` command, to pre-filter the results?